### ISSUE TITLE
Hugging face notebook fail

### ISSUE THREAD

**Issue by tedhtchang** [project-codeflare/codeflare-sdk#170]
The [hf_interactive](https://github.com/project-codeflare/codeflare-sdk/blob/main/demo-notebooks/interactive/hf_interactive.ipynb) notebook (train_fn.remote()) failed with the following message:
```
ImportError: Datasets requires pyarrow >= 6.0.1, < 7.0.0, but 12.0.1 is installed. Reinstall with `pip install -U "pyarrow<7.0.0"`. If you want to disable this pyarrow version check, set the environment variable RAY_DISABLE_PYARROW_VERSION_CHECK=1.
```

Edited:
Got it to pass the failure with
`runtime_env = {"pip": ["transformers", "datasets", "evaluate","pyarrow<7.0.0","transformers[torch]"]}`
but I don't think I can get away with no gpu.
```
RuntimeError: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx
```

**Comment by anishasthana** [comment]
Completed as per @MichaelClifford

**Comment by MichaelClifford** [comment]
This error happened when the `runtime_env` was set incorrectly in the notebook. This has been addressed in #163 and should no longer be an issue
